Running in wrapper mode for '222'

Running python job.

[name: "CPU-QUOTA"
int_val: 82
int_val: 11
int_val: 80
int_val: 6
int_val: 91
int_val: 100
int_val: 30
, name: "DISK"
int_val: 40
int_val: 39
int_val: 39
int_val: 7
int_val: 28
int_val: 32
int_val: 35
, name: "MEMORY"
int_val: 34
int_val: 37
int_val: 10
int_val: 9
int_val: 17
int_val: 18
int_val: 5
, name: "NET"
int_val: 30
int_val: 36
int_val: 8
int_val: 11
int_val: 6
int_val: 8
int_val: 30
]
The paramters are {u'CPU-QUOTA': array([ 82,  11,  80,   6,  91, 100,  30]), u'NET': array([30, 36,  8, 11,  6,  8, 30]), u'DISK': array([40, 39, 39,  7, 28, 32, 35]), u'MEMORY': array([34, 37, 10,  9, 17, 18,  5])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 27500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 184320.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 32715571.2
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 1387500000.0
Recovering the memory utilization
Setting cpu quota to 205000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 153600.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 33554432.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 1275000000.0
Recovering the memory utilization
Setting cpu quota to 200000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 40960.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 32715571.2
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 375000000.0
Recovering the memory utilization
Setting cpu quota to 205000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 153600.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 33554432.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 1275000000.0
Recovering the memory utilization
Setting cpu quota to 200000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 40960.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 32715571.2
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 375000000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7f4038482710>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.19 seconds.
